testdata/BUILD

```diff
@@ -938,3 +938,22 @@ go_binary(
     name = "main",
     srcs = ["main.go"],
 )
+
+container_image(
```

Comment:
Looks good! Could you also add a test case the default one is used when docker_run_flags is not specified?